filters/f_swscale.c

```diff
@@ -106,12 +106,6 @@ static void process(struct mp_filter *f)
 
     mp_image_copy_attributes(dst, src);
 
-    // If we convert from RGB to YUV, guess a default.
-    if (mp_imgfmt_get_forced_csp(src->imgfmt) == MP_CSP_RGB &&
-        mp_imgfmt_get_forced_csp(dst->imgfmt) == MP_CSP_AUTO)
-    {
-        dst->params.color.levels = MP_CSP_LEVELS_AUTO;
```

Comment:
Why is this change needed? Can you add at least a commit message explaining the justification?

Reply:
I will update it after work, I think I should remove it, because it is not related to the PR. Although I don't see a reason to not output full range.

Reply:
updated, I left this change in separate commit. I really don't like resetting color params like that, without clear explanation why.